### DISCUSSION TITLE
About the sart3d method of cone beam

### DISCUSSION THREAD

**Discussion by PeterPaang4** [astra-toolbox/astra-toolbox#505] [Q&A]
Hello, dear author, I need to implement the sart3d function about cone beams, but what I found on astra-toolbox is the sart-2d method. Do you have any suggestions? Thank you.

**Comment by askorikov** [comment]
Hi, I believe you can implement it from forward and backprojections + some math. You can draw inspiration from the SIRT implementation here: https://github.com/astra-toolbox/astra-toolbox/blob/2ff4ebc17f22dd34aa620d405358e02020012eec/python/astra/plugins/sirt.py#L77-L88 However, for SART you will need to construct new `W`, `R`, and `mrC` objects corresponding to different sets of projection angles at each iteration.
